.pipelines/templates/.builder-release-template.yaml

```diff
@@ -17,6 +17,22 @@ steps:
       vstsFeedPackage: asc-baseline
       vstsPackageVersion: '1.1.0-268'
       downloadDirectory: vhdbuilder/packer
+  - task: DownloadPipelineArtifact@2
+    displayName: 'Download Kata CC UVM artifact'
+    inputs:
+      source: specific
+      project: CloudNativeCompute
+      pipeline: $(UVM_PIPELINE_ID)
+      runVersion: specific
+      runId: $(UVM_RUN_ID)
+      artifactName: 'drop_build_and_sign_build_and_sign'
+      downloadType: 'single'
+      itemPattern: |
+        **/kata-containers**
+        **/igvm**
+        **/cloud-hypervisor**
```

Comment:
The cloud-hypervisor binary is also a temporary artifact until upstreamed by LSG